man/resolved.conf.xml

```diff
@@ -335,6 +335,13 @@ DNSStubListenerExtra=udp:[2001:db8:0:f102::13]:9953</programlisting>
         url="https://www.iab.org/documents/correspondence-reports-documents/2013-2/iab-statement-dotless-domains-considered-harmful/">IAB
         Statement</ulink>, and may create a privacy and security risk.</para></listitem>
       </varlistentry>
+      <varlistentry>
+        <term>StaleRetentionSec=<replaceable>SECONDS</replaceable></term>
+        <listitem><para>Takes a duration value to determine for how long DNS resource records can be retained in the cache beyond their Time To Live (TTL) to return them as stale records.
+        This is useful when a DNS server failure occurs or not reachable, as systemd-resolved will continue to use these stale records to answer the DNS questions in case of a negative response from the upstream servers.
+        This improves the resilience against DNS infrastructure failures and outages.
+        However, this doesn't apply to NXDOMAIN responses, as those are still perfectly valid responses.</para></listitem>
+      </varlistentry>
```

Comment:
event though the above kinda says this already I think it would make sense to say expressly in a separate sentence that before resolved returns stale data it will always try to contact the upstream DNS servers first.

Reply:
Added